class_incremets.py

```diff
@@ -0,0 +1,41 @@
+class Car:
+    """
+    This is a car class
+    """
+    nrOfInst = 0
```

Comment:
Use the function naming rules: lowercase with words separated by underscores as necessary to improve readability.
https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables